balboa-agent/src/main/java/com/socrata/balboa/agent/HttpOrMq.java

```diff
@@ -0,0 +1,19 @@
+package com.socrata.balboa.agent;
+
+public enum HttpOrMq {
```

Comment:
`TransportType` would be a better name. Supports the expansion, or contraction, of the enumeration in the future.